api/v2/endpoints/post-identity-map.md

```diff
@@ -38,7 +38,7 @@ Here's what you need to know:
 | `email_hash` | string array | Conditionally Required | The list of [Base64-encoded SHA-256](../../README.md#email-address-hash-encoding) hashes of [normalized](../../README.md#email-address-normalization) email addresses. |
 | `phone` | string array | Conditionally Required | The list of [normalized](../../README.md#phone-number-normalization) phone numbers to be mapped. |
 | `phone_hash` | string array | Conditionally Required | The list of [Base64-encoded SHA-256](../../README.md#email-address-hash-encoding) hashes of [normalized](../../README.md#phone-number-normalization) phone numbers. |
-
+| `policy` | integer | Optional | Customize the identity mapping behavior when a user identifier is opted out. See [Identity Map Policy](#identity-map-policy) |
```

Comment:
| `policy` | integer | Optional | Customize the identity mapping behavior when a user identifier is opted out. For details, see [Identity Map Policy](#identity-map-policy). |

Reply:
great, changed!